timesketch/api/v1/resources/analysis.py

```diff
@@ -32,6 +34,9 @@
 from timesketch.models.sketch import Timeline
 
 
+logger = logging.getLogger('timesketch.sketch_api')
```

Comment:
this shouldn't be the `sketch_api`, this is `analysis_api`

Reply:
done